blog-posts/howto-blog-post.md

```diff
@@ -53,7 +53,25 @@ Here a 4MB picture that should get prepared by gatsby properly.
 We don't want to load 4MB but the processed image:
 
 ![twilight coloured imaginary planet where the surface shows some pink crystals growing and three planets visible in the sky](./images/space.jpg)
-<a href='https://www.freepik.com/vectors/star'>Star vector created by upklyak</a>
+*[Star vector created by upklyak](https://www.freepik.com/vectors/star)*
+
+#### Image caption
+
+To maintain a valid HTML image structure and an easily readable markdown-file, the caption for the image can be added like this:
+```
+![alt-text](./image-url)
+*Picture 1: This is an example caption.*
+```
```

Comment:
this is different to the billing dashboard example where the text is just added as a paragraph?